### ISSUE TITLE
[Bug]: pkgcheck doesn't complain if `<remote-id>` contains tabs or newlines

### ISSUE THREAD

**Issue by emanuele6** [pkgcore/pkgcheck#697]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

If you have something like
```xml
<remote-id type="github">
	user/repo
</remote-id>
```
`pkgcheck scan` doesn't complain about the TAB and newline characters in `<remote-id>`.

If it is indented spaces instead of TABs, it will complain with BadUrl because it will check `https://github.com/%20%20%20%20%20%20%20%20user/repo`, but TABs and newlines are completely ignored.

### Expected Behavior

It should complain if `<remote-id>` contains TABs and/or newlines.

https://github.com/gentoo/gentoo/pull/38258#issuecomment-2308348542
https://bugs.gentoo.org/938339

### pkgcheck version

pkgcheck 0.10.30

### pkgcore version

pkgcore 0.12.27

**Comment by arthurzam** [comment]
I've fixed the issue on packages.gentoo.org side, and I think this is correct behaviour, since whitespaces are allowed in XML on both sides.
